### PR TITLE
refactor: use new colors Notice, PasswordCheck,PasswordStrengthMeter, Pentagon, PhoneInput, PieChart

### PR DESCRIPTION
**Pull request by adriengibrat**
use new colors in Notice, PasswordCheck,PasswordStrengthMeter, Pentagon, PhoneInput, PieChart

## Type

- Migration
- Refactor

#### What is expected?

No major update, some grey level updated.

## Relevant screenshots

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| Notice  |image |image |
|PasswordCheck|image|image|
| PasswordStrengthMeter  | image | image |
| Pentagon  | image | image |
| PhoneInput  | image | image |
| PieChart  |image | image|

### CODE REVIEW

File: src/components/PasswordCheck/index.tsx [hunk 1]
```diff
@@ -31,27 +31,35 @@ type PasswordCheckProps = {
...
+        {rules.map(rule => (
+          <Col large={6} key={rule.name}>
+            <p css={styles.condition}>
+              <Icon
+                name={
+                  rule.valid ? 'check-circle-outline' : 'close-circle-outline'
+                }
+                color={
+                  rule.valid
+                    ? theme.colors.success.text
```
Comment (matthprost) [inline src/components/PasswordCheck/index.tsx:49]:
```suggestion
                    ? theme.colors.success.textWeak
```

We are in a weak context (background is white), even if color is same now :)

File: src/components/PasswordStrengthMeter/index.tsx [hunk 2]
```diff
@@ -66,8 +67,9 @@ const PasswordStrengthMeter = ({
   userInputs = [],
 }: PasswordStrengthMeterProps): JSX.Element => {
   const [score, setScore] = useState<number>(0)
+  const theme = useTheme()
   const [backgroundColor, setBackgroundColor] = useState<string>(
-    strength[0]?.color || 'green',
+    strength[0]?.color || theme.colors.success.background,
```
Comment (matthprost) [inline src/components/PasswordStrengthMeter/index.tsx:72]:
```suggestion
    strength[0]?.color || theme.colors.success.backgroundStrong,
```

Should be better no ? As default background is really light

File: src/components/PhoneInput/index.tsx [hunk 3]
```diff
@@ -35,8 +34,7 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
   position: relative;
   display: flex;
   border-radius: 4px;
-  border: 1px solid
-    ${({ theme: { colorsDeprecated } }) => colorsDeprecated.gray350};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.border};
```
Comment (matthprost) [inline src/components/PhoneInput/index.tsx:37]:
```suggestion
  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
```

Same here background is white so we are in a weak context 😄

File: src/components/PhoneInput/index.tsx [hunk 4]
```diff
@@ -45,11 +43,10 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
   }
 
   &:focus-within {
-    border: 1px solid
-      ${({ theme: { colorsDeprecated } }) => colorsDeprecated.primary};
+    border: 1px solid ${({ theme }) => theme.colors.primary.borderStrong};
```
Comment (matthprost) [inline src/components/PhoneInput/index.tsx:46]: Here too you can make it weak to follow context

```suggestion
    border: 1px solid ${({ theme }) => theme.colors.primary.borderWeak};
```

File: src/components/PieChart/index.tsx [hunk 5]
```diff
@@ -117,7 +117,7 @@ const PieChart: VoidFunctionComponent<PieChartProps> = ({
           defs={[
             {
               background: 'inherit',
-              color: colorsDeprecated.white,
+              color: colors.neutral.textWeakHover,
```
Comment (matthprost) [inline src/components/PieChart/index.tsx:120]:
```suggestion
              color: colors.neutral.textStrong,
```

With strong background context text should be white in same context and will fix this difference:

Screenshot 2021-12-14 at 12 29 13
Screenshot 2021-12-14 at 12 29 18

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by chambo-e — APPROVED** [review]
This also need a more detailed PR title :)

File: src/components/PasswordCheck/index.tsx (hunk 1 above)
Comment (adriengibrat) [reply src/components/PasswordCheck/index.tsx:49]: done

File: src/components/PasswordStrengthMeter/index.tsx (hunk 2 above)
Comment (adriengibrat) [reply src/components/PasswordStrengthMeter/index.tsx:72]: yes, done

File: src/components/PhoneInput/index.tsx (hunk 3 above)
Comment (adriengibrat) [reply src/components/PhoneInput/index.tsx:37]: done

File: src/components/PhoneInput/index.tsx (hunk 4 above)
Comment (adriengibrat) [reply src/components/PhoneInput/index.tsx:46]: done

File: src/components/PieChart/index.tsx (hunk 5 above)
Comment (adriengibrat) [reply src/components/PieChart/index.tsx:120]: oops, done

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/components/PasswordCheck/index.tsx (hunk 1 above)
Comment (adriengibrat) [reply src/components/PasswordCheck/index.tsx:49]: yep, updated the neutral but not the success... sorry, done!

**Review by matthprost — APPROVED** [review]

**Pull request merged**